.github/workflows/deploy.yml

```diff
@@ -42,7 +42,10 @@ jobs:
           bun run sst secret set StackProjectId ${{ secrets.STACK_PROJECT_ID }} --stage production
           bun run sst secret set StackPublishableClientKey ${{ secrets.STACK_PUBLISHABLE_CLIENT_KEY }} --stage production
           bun run sst secret set StackSecretServerKey ${{ secrets.STACK_SECRET_SERVER_KEY }} --stage production
-          bun run sst secret set GroqApiKey ${{ secrets.GROQ_API_KEY }} --stage production
+          bun run sst secret set BedrockRoleArn ${{ secrets.BEDROCK_ROLE_ARN }} --stage production
+          bun run sst secret set GoogleVertexServiceAccountEmail ${{ secrets.GOOGLE_VERTEX_SERVICE_ACCOUNT_EMAIL }} --stage production
+          bun run sst secret set GoogleVertexAwsProviderAudience ${{ secrets.GOOGLE_VERTEX_AWS_PROVIDER_AUDIENCE }} --stage production
```

Comment:
If I understand correctly, this is only required for provider forwarding. Therefore it shouldn't be required as a deploy time secret?